standalone-metastore/metastore-common/pom.xml

```diff
@@ -445,21 +436,6 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-antrun-plugin</artifactId>
         <executions>
-          <execution>
```

Comment:
`metastore-common` provides the common utils and apis for metastore, and we have only one test class in this module: [RetryTest](https://github.com/apache/hive/blob/master/standalone-metastore/metastore-common/src/test/java/org/apache/hadoop/hive/metastore/utils/RetryTest.java), this can be run without any requirements.